pkg/util/http_test.go

```diff
@@ -147,55 +147,67 @@ func TestStreamWriteYAMLResponse(t *testing.T) {
 
 func TestParseProtoReader(t *testing.T) {
 	// 47 bytes compressed and 53 uncompressed
-	req := &mimirpb.PreallocWriteRequest{
-		WriteRequest: mimirpb.WriteRequest{
-			Timeseries: []mimirpb.PreallocTimeseries{
-				{
-					TimeSeries: &mimirpb.TimeSeries{
-						Labels: []mimirpb.LabelAdapter{
-							{Name: "foo", Value: "bar"},
-						},
-						Samples: []mimirpb.Sample{
-							{Value: 10, TimestampMs: 1},
-							{Value: 20, TimestampMs: 2},
-							{Value: 30, TimestampMs: 3},
-						},
-						Exemplars: []mimirpb.Exemplar{},
-					},
+	req := &prompb.WriteRequest{
+		Timeseries: []prompb.TimeSeries{
+			{
+				Labels: []prompb.Label{
+					{Name: "foo", Value: "bar"},
+				},
+				Samples: []prompb.Sample{
+					{Value: 10, Timestamp: 1},
+					{Value: 20, Timestamp: 2},
+					{Value: 30, Timestamp: 3},
 				},
 			},
 		},
 	}
 
 	for _, tt := range []struct {
-		name           string
-		compression    util.CompressionType
-		maxSize        int
-		expectErr      bool
-		useBytesBuffer bool
+		name                  string
+		compression           CompressionType
+		maxSize               int
+		expectSerializeErr    bool
+		expectParseErr        bool
+		useBytesBuffer        bool
+		mockTooLargeForSnappy bool
```

Comment:
is this set as `true` in any test case?

Reply:
Yes, in the `"too big encoded rawSnappy"` test